apps/DemoApp/DemoApp/ViewController.m

```diff
@@ -318,7 +318,16 @@ - (void)playerItemDidReachEnd:(NSNotification *)notification {
     MUXSDKCustomerVideoData *videoData = [MUXSDKCustomerVideoData new];
     videoData.videoTitle = @"Apple Keynote";
     videoData.videoId = @"applekeynote2010";
-    [MUXSDKStats videoChangeForPlayer:DEMO_PLAYER_NAME withVideoData:videoData];
+    
+    MUXSDKCustomerData *customerData = [
```

Comment:
This is unrelated but it was to get rid of the notification for using the deprecated videoChange function